x-pack/plugin/security/cli/build.gradle

```diff
@@ -31,11 +32,11 @@ tasks.named("forbiddenPatterns").configure {
 }
 
 tasks.named("thirdPartyAudit").configure {
-  ignoreMissingClasses(
-    // Used in org.bouncycastle.pqc.crypto.qtesla.QTeslaKeyEncodingTests
-    'junit.framework.Assert',
-    'junit.framework.TestCase'
-  )
+  ignoreMissingClasses()
+}
+
+tasks.named("test").configure {
+  systemProperty 'tests.security.manager', 'false' // the main code under test runs without the SecurityManager
```

Comment:
this already exists in 8.x ... the updated jar requires a security privilege, but since we don't run this cli with the security manager, just disable it for the test